docs/src/main/sphinx/release/release-443.md

```diff
@@ -0,0 +1,46 @@
+# Release 443 (21 Mar 2024)
+
+## General
+
+* Ensure consistent formatting when casting from double or real to varchar. ({issue}`21136`)
+* Prevent query failure when partial aggregation over long decimals is pushed
+  below joins. ({issue}`21099`)
+
+## Delta Lake connector
+
+* Support automatic type coercion for timestamp type columns during in table
+  creation. ({issue}`19336`)
+* Ensure deletion of blobs ending in slash when deleting directory contents and
+  using the native S3 file system support. ({issue}`21111`)
+
+## Hive connector
+
+* Reduce coordinator CPU and memory usage. ({issue}`21075`)
+* Prevent failures when listing columms of a table that is concurrently dropped
+  and `sql-standard` authorization is used. ({issue}`21109`)
+* Ensure deletion of blobs ending in slash when deleting directory contents and
```

Comment:
This seems like an implementation detail. Under what scenarios would those blobs not have been deleted? 

I would rephrase as: "Ensure all files are deleted when <user triggered action> and native S3 filesystem support is enabled"

Reply:
There was another bug about paths with and without ending in slash being treated differently.